app/views/course/announcements/index.json.jbuilder

```diff
@@ -0,0 +1,26 @@
+json.announcements @announcements do |announcement| 
+  json.title announcement.title
```

Comment:
Looks nearly identical to `announcement.json.jbuilder`. Can it be refactored into a partial?